File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/OccupancyRateAmount.vb
Imports Newtonsoft.Json
Namespace ExpediaRapid.SerializableClasses

    Public Class OccupancyRateAmount

        <JsonProperty("value")>
        Public Property Amount As Decimal

        <JsonProperty("currency")>
        Public Property CurrencyCode As String

    End Class

End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/OccupancyRateAmount.cs
namespace iVectorOne.Suppliers.ExpediaRapid.SerializableClasses
{
    using Newtonsoft.Json;

    public class OccupancyRateAmount
    {

        [JsonProperty("value")]
        public decimal Amount { get; set; }

        [JsonProperty("currency")]
        public string CurrencyCode { get; set; }

    }

}